docs/package-lock.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
I am surprised to see a lock file here but I guess this is just what the docs site requires?

Reply:
Yes it uses npm.

Reply:
I see that on other docs sites repos too: 
- https://github.com/helm/helm-www/blob/master/package.json
- https://github.com/servicemeshinterface/smi-spec.io/blob/main/package.json 

Probably we just usually don't notice it for the sites where the content is in the web repo?